src/main/java/frc/robot/commands/ScoreNoteCommand.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ScoringConstants;
+import frc.robot.subsystems.Scoring;
+
+public class ScoreNoteCommand extends Command {
+    private Scoring scoring = Scoring.getInstance();
+    private Timer timer = new Timer();
+
+    public ScoreNoteCommand() {
+        this.addRequirements(scoring);
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        scoring.runRoller(0);
```

Comment:
we could replace the speed parameter with a default speed

Reply:
This should at least be a constant